spec/api-debugger-spec.js

```diff
@@ -123,7 +138,24 @@ describe('debugger module', () => {
       w.webContents.debugger.sendCommand('Runtime.evaluate', params, callback)
     })
 
-    it('returns response when devtools is opened', done => {
+    it('returns response when devtools is opened', async () => {
+      w.webContents.loadURL('about:blank')
+      w.webContents.debugger.attach()
+
+      w.webContents.openDevTools()
+      await emittedOnce(w.webContents, 'devtools-opened')
```

Comment:
🚨 FLAKE ALERT 🚨 

you need to call `emittedOnce` _before_ you call `openDevTools`. As written this is a race condition and will result in a flaky test.